driver/task/manager.cpp

```diff
@@ -94,6 +94,7 @@ freighter::Error task::Manager::stop() {
     streamer->closeSend();
     run_thread.join();
     for (auto &[key, task]: tasks) task->stop();
+    breaker.close();
```

Comment:
I feel like we're using several different mechanisms to shut things down. This is confusing, and will make it hard to reliably implement startup and shutdown logic for new tasks. We need to simplify